2024-08-10 19:44:38,199 - /home/varsha/Documents/Projects/nextgrowth/app_point_system/app_point_system/middleware.py - 23 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 18
2024-08-10 19:46:01,549 - /home/varsha/Documents/Projects/nextgrowth/app_point_system/app_point_system/middleware.py - 23 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 18
2024-08-10 19:47:21,360 - /home/varsha/Documents/Projects/nextgrowth/app_point_system/app_point_system/middleware.py - 23 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 18
2024-08-10 19:49:50,709 - /home/varsha/Documents/Projects/nextgrowth/app_point_system/app_point_system/middleware.py - 23 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 18
2024-08-10 19:53:34,397 - /home/varsha/Documents/Projects/nextgrowth/app_point_system/app_point_system/middleware.py - 23 - middleware - __call__ - ERROR - 'HttpResponseNotFound' object has no attribute 'data' - line no: 18
2024-08-11 20:32:46,080 - /home/varsha/Documents/Projects/nextgrowth/app_point_system/app_point_system/middleware.py - 23 - middleware - __call__ - ERROR - 'HttpResponseNotFound' object has no attribute 'data' - line no: 18
2024-08-11 20:32:46,245 - /home/varsha/Documents/Projects/nextgrowth/app_point_system/app_point_system/middleware.py - 23 - middleware - __call__ - ERROR - 'HttpResponseNotFound' object has no attribute 'data' - line no: 18
2024-08-11 20:32:58,200 - /home/varsha/Documents/Projects/nextgrowth/app_point_system/app_point_system/middleware.py - 18 - middleware - __call__ - ERROR - {'detail': ErrorDetail(string='Method "GET" not allowed.', code='method_not_allowed')} - No traceback available
2024-08-11 20:47:08,100 - /home/varsha/Documents/Projects/nextgrowth/app_point_system/app_point_system/middleware.py - 23 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 18
2024-08-11 20:47:25,896 - /home/varsha/Documents/Projects/nextgrowth/app_point_system/app_point_system/middleware.py - 23 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 18
2024-08-11 20:48:28,677 - /home/varsha/Documents/Projects/nextgrowth/app_point_system/app_point_system/middleware.py - 23 - middleware - __call__ - ERROR - 'HttpResponseNotFound' object has no attribute 'data' - line no: 18
2024-08-11 20:48:33,368 - /home/varsha/Documents/Projects/nextgrowth/app_point_system/app_point_system/middleware.py - 23 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 18
2024-08-11 20:49:52,357 - /home/varsha/Documents/Projects/nextgrowth/app_point_system/app_point_system/middleware.py - 23 - middleware - __call__ - ERROR - 'HttpResponseNotFound' object has no attribute 'data' - line no: 18
2024-08-11 20:50:00,311 - /home/varsha/Documents/Projects/nextgrowth/app_point_system/app_point_system/middleware.py - 23 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 18
2024-08-11 20:51:15,841 - /home/varsha/Documents/Projects/nextgrowth/app_point_system/app_point_system/middleware.py - 23 - middleware - __call__ - ERROR - 'HttpResponseNotFound' object has no attribute 'data' - line no: 18
2024-08-11 20:51:21,834 - /home/varsha/Documents/Projects/nextgrowth/app_point_system/app_point_system/middleware.py - 23 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 18
2024-08-11 20:51:35,262 - /home/varsha/Documents/Projects/nextgrowth/app_point_system/app_point_system/middleware.py - 23 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 18
2024-08-11 20:51:54,017 - /home/varsha/Documents/Projects/nextgrowth/app_point_system/app_point_system/middleware.py - 23 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 18
2024-08-11 20:51:57,008 - /home/varsha/Documents/Projects/nextgrowth/app_point_system/app_point_system/middleware.py - 23 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 18
2024-08-11 21:39:28,662 - /home/varsha/Documents/Projects/nextgrowth/app_point_system/app_point_system/middleware.py - 18 - middleware - __call__ - ERROR - {'success': False, 'details': 'Invalid credentials'} - No traceback available
2024-08-11 21:55:54,113 - /home/varsha/Documents/Projects/nextgrowth/app_point_system/app_point_system/middleware.py - 18 - middleware - __call__ - ERROR - {'success': False, 'details': {'email': [ErrorDetail(string='This field is required.', code='required')]}} - No traceback available
2024-08-11 21:58:18,197 - /home/varsha/Documents/Projects/nextgrowth/app_point_system/app_point_system/middleware.py - 23 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 18
2024-08-11 22:00:47,859 - /home/varsha/Documents/Projects/nextgrowth/app_point_system/app_point_system/middleware.py - 23 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 18
2024-08-11 22:01:36,225 - /home/varsha/Documents/Projects/nextgrowth/app_point_system/app_point_system/middleware.py - 18 - middleware - __call__ - ERROR - {'success': False, 'details': {'non_field_errors': [ErrorDetail(string='Email already exists', code='invalid')]}} - No traceback available
2024-08-11 22:23:57,036 - /home/varsha/Documents/Projects/nextgrowth/app_point_system/app_point_system/middleware.py - 23 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 18
2024-08-11 22:24:12,200 - /home/varsha/Documents/Projects/nextgrowth/app_point_system/app_point_system/middleware.py - 23 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 18
2024-08-11 22:26:08,608 - /home/varsha/Documents/Projects/nextgrowth/app_point_system/app_point_system/middleware.py - 23 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 18
2024-08-12 10:45:45,087 - /home/varsha/Documents/Projects/nextgrowth/app_point_system/app_point_system/middleware.py - 23 - middleware - __call__ - ERROR - 'HttpResponseNotFound' object has no attribute 'data' - line no: 18
2024-08-12 11:01:29,698 - /home/varsha/Documents/Projects/nextgrowth/app_point_system/app_point_system/middleware.py - 23 - middleware - __call__ - ERROR - 'HttpResponseNotFound' object has no attribute 'data' - line no: 18
2024-08-12 11:55:39,476 - /home/varsha/Documents/Projects/nextgrowth/app_point_system/app_point_system/middleware.py - 18 - middleware - __call__ - ERROR - {'detail': ErrorDetail(string='Authentication credentials were not provided.', code='not_authenticated')} - No traceback available
2024-08-12 12:35:11,539 - /home/varsha/Documents/Projects/nextgrowth/app_point_system/app_point_system/middleware.py - 18 - middleware - __call__ - ERROR - {'detail': ErrorDetail(string='Authentication credentials were not provided.', code='not_authenticated')} - No traceback available
2024-08-12 12:36:58,893 - /home/varsha/Documents/Projects/nextgrowth/app_point_system/app_point_system/middleware.py - 23 - middleware - __call__ - ERROR - 'HttpResponseNotFound' object has no attribute 'data' - line no: 18
2024-08-12 12:37:03,426 - /home/varsha/Documents/Projects/nextgrowth/app_point_system/app_point_system/middleware.py - 23 - middleware - __call__ - ERROR - 'HttpResponseNotFound' object has no attribute 'data' - line no: 18
2024-08-12 12:40:42,306 - /home/varsha/Documents/Projects/nextgrowth/app_point_system/app_point_system/middleware.py - 18 - middleware - __call__ - ERROR - {'detail': ErrorDetail(string='Given token not valid for any token type', code='token_not_valid'), 'code': ErrorDetail(string='token_not_valid', code='token_not_valid'), 'messages': [{'token_class': ErrorDetail(string='AccessToken', code='token_not_valid'), 'token_type': ErrorDetail(string='access', code='token_not_valid'), 'message': ErrorDetail(string='Token is invalid or expired', code='token_not_valid')}]} - No traceback available
2024-08-12 12:52:37,332 - /home/varsha/Documents/Projects/nextgrowth/app_point_system/app_point_system/middleware.py - 18 - middleware - __call__ - ERROR - {'detail': ErrorDetail(string='Given token not valid for any token type', code='token_not_valid'), 'code': ErrorDetail(string='token_not_valid', code='token_not_valid'), 'messages': [{'token_class': ErrorDetail(string='AccessToken', code='token_not_valid'), 'token_type': ErrorDetail(string='access', code='token_not_valid'), 'message': ErrorDetail(string='Token is invalid or expired', code='token_not_valid')}]} - No traceback available
2024-08-12 12:53:32,481 - /home/varsha/Documents/Projects/nextgrowth/app_point_system/app_point_system/middleware.py - 18 - middleware - __call__ - ERROR - {'detail': ErrorDetail(string='Given token not valid for any token type', code='token_not_valid'), 'code': ErrorDetail(string='token_not_valid', code='token_not_valid'), 'messages': [{'token_class': ErrorDetail(string='AccessToken', code='token_not_valid'), 'token_type': ErrorDetail(string='access', code='token_not_valid'), 'message': ErrorDetail(string='Token is invalid or expired', code='token_not_valid')}]} - No traceback available
2024-08-12 12:54:56,306 - /home/varsha/Documents/Projects/nextgrowth/app_point_system/app_point_system/middleware.py - 18 - middleware - __call__ - ERROR - {'detail': ErrorDetail(string='Given token not valid for any token type', code='token_not_valid'), 'code': ErrorDetail(string='token_not_valid', code='token_not_valid'), 'messages': [{'token_class': ErrorDetail(string='AccessToken', code='token_not_valid'), 'token_type': ErrorDetail(string='access', code='token_not_valid'), 'message': ErrorDetail(string='Token is invalid or expired', code='token_not_valid')}]} - No traceback available
2024-08-12 12:56:34,683 - /home/varsha/Documents/Projects/nextgrowth/app_point_system/app_point_system/middleware.py - 18 - middleware - __call__ - ERROR - {'detail': ErrorDetail(string='Authentication credentials were not provided.', code='not_authenticated')} - No traceback available
2024-08-12 12:58:10,703 - /home/varsha/Documents/Projects/nextgrowth/app_point_system/app_point_system/middleware.py - 18 - middleware - __call__ - ERROR - {'detail': ErrorDetail(string='Authentication credentials were not provided.', code='not_authenticated')} - No traceback available
2024-08-12 12:58:21,858 - /home/varsha/Documents/Projects/nextgrowth/app_point_system/app_point_system/middleware.py - 18 - middleware - __call__ - ERROR - {'detail': ErrorDetail(string='Authentication credentials were not provided.', code='not_authenticated')} - No traceback available
2024-08-12 13:04:13,640 - /home/varsha/Documents/Projects/nextgrowth/app_point_system/app_point_system/middleware.py - 18 - middleware - __call__ - ERROR - {'detail': ErrorDetail(string='Authentication credentials were not provided.', code='not_authenticated')} - No traceback available
2024-08-12 13:08:42,043 - /home/varsha/Documents/Projects/nextgrowth/app_point_system/app_point_system/middleware.py - 18 - middleware - __call__ - ERROR - {'detail': ErrorDetail(string='Authentication credentials were not provided.', code='not_authenticated')} - No traceback available
2024-08-12 13:35:39,726 - /home/varsha/Documents/Projects/nextgrowth/app_point_system/app_point_system/middleware.py - 18 - middleware - __call__ - ERROR - {'detail': ErrorDetail(string='Method "GET" not allowed.', code='method_not_allowed')} - No traceback available
2024-08-12 20:39:50,600 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 23 - middleware - __call__ - ERROR - 'HttpResponseNotFound' object has no attribute 'data' - line no: 18
2024-08-12 20:39:50,765 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 23 - middleware - __call__ - ERROR - 'HttpResponseNotFound' object has no attribute 'data' - line no: 18
2024-08-12 20:48:09,219 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 23 - middleware - __call__ - ERROR - 'HttpResponseNotFound' object has no attribute 'data' - line no: 18
2024-08-12 20:48:09,496 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 23 - middleware - __call__ - ERROR - 'HttpResponseNotFound' object has no attribute 'data' - line no: 18
2024-08-12 20:48:19,047 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 23 - middleware - __call__ - ERROR - 'HttpResponseNotFound' object has no attribute 'data' - line no: 18
2024-08-12 20:48:27,053 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 18 - middleware - __call__ - ERROR - {'detail': ErrorDetail(string='Method "GET" not allowed.', code='method_not_allowed')} - No traceback available
2024-08-12 20:48:52,444 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 23 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 18
2024-08-12 21:35:57,211 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 18 - middleware - __call__ - ERROR - {'detail': ErrorDetail(string='Method "GET" not allowed.', code='method_not_allowed')} - No traceback available
2024-08-12 21:36:45,313 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 23 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 18
2024-08-12 21:37:29,860 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 18 - middleware - __call__ - ERROR - {'detail': ErrorDetail(string='Authentication credentials were not provided.', code='not_authenticated')} - No traceback available
2024-08-12 21:59:44,420 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponseNotFound' object has no attribute 'data' - line no: 30
2024-08-12 21:59:52,649 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponseNotFound' object has no attribute 'data' - line no: 30
2024-08-13 12:02:08,169 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponseNotFound' object has no attribute 'data' - line no: 30
2024-08-13 12:02:08,390 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponseNotFound' object has no attribute 'data' - line no: 30
2024-08-13 12:02:23,911 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponseNotFound' object has no attribute 'data' - line no: 30
2024-08-13 12:02:57,293 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 30
2024-08-13 12:03:32,013 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 30 - middleware - __call__ - ERROR - {'detail': ErrorDetail(string='Authentication credentials were not provided.', code='not_authenticated')} - No traceback available
2024-08-13 13:20:41,776 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponseNotFound' object has no attribute 'data' - line no: 30
2024-08-13 13:20:53,767 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 30
2024-08-13 14:23:46,894 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 30
2024-08-13 14:23:51,093 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 30
2024-08-13 14:25:01,632 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 30
2024-08-13 14:34:05,475 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 30
2024-08-13 14:34:45,790 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 30
2024-08-13 14:37:56,848 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 30
2024-08-13 14:38:45,847 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 30
2024-08-13 15:34:43,075 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponseNotFound' object has no attribute 'data' - line no: 30
2024-08-13 15:39:46,367 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 30
2024-08-13 15:39:52,767 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 30
2024-08-13 17:34:17,029 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponseNotFound' object has no attribute 'data' - line no: 30
2024-08-13 17:37:03,101 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponseForbidden' object has no attribute 'data' - line no: 30
2024-08-13 17:37:34,450 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponseForbidden' object has no attribute 'data' - line no: 30
2024-08-13 17:38:01,156 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 30 - middleware - __call__ - ERROR - {'detail': ErrorDetail(string='Authentication credentials were not provided.', code='not_authenticated')} - No traceback available
2024-08-13 19:46:46,844 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 30
2024-08-13 19:46:47,100 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponseNotFound' object has no attribute 'data' - line no: 30
2024-08-13 19:47:27,666 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 30
2024-08-13 20:01:15,776 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponseNotFound' object has no attribute 'data' - line no: 30
2024-08-13 20:01:26,067 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponseNotFound' object has no attribute 'data' - line no: 30
2024-08-13 20:07:54,913 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponseNotFound' object has no attribute 'data' - line no: 30
2024-08-13 20:09:38,623 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 30 - middleware - __call__ - ERROR - {'success': False, 'details': {'name': [ErrorDetail(string='This field may not be blank.', code='blank')]}} - No traceback available
2024-08-13 20:09:50,771 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 30 - middleware - __call__ - ERROR - {'success': False, 'details': {'name': [ErrorDetail(string='This field may not be blank.', code='blank')]}} - No traceback available
2024-08-13 20:09:52,952 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 30 - middleware - __call__ - ERROR - {'success': False, 'details': {'name': [ErrorDetail(string='This field is required.', code='required')]}} - No traceback available
2024-08-13 20:09:54,440 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 30 - middleware - __call__ - ERROR - {'success': False, 'details': {'name': [ErrorDetail(string='This field is required.', code='required')]}} - No traceback available
2024-08-13 20:09:58,016 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 30 - middleware - __call__ - ERROR - {'success': False, 'details': {'name': [ErrorDetail(string='This field may not be blank.', code='blank')]}} - No traceback available
2024-08-13 20:10:15,093 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 30
2024-08-13 20:10:35,260 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 30
2024-08-13 20:16:52,793 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 30 - middleware - __call__ - ERROR - {'success': False, 'details': {'name': [ErrorDetail(string='This field may not be blank.', code='blank')]}} - No traceback available
2024-08-13 20:17:04,644 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 30
2024-08-13 20:17:48,659 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 30 - middleware - __call__ - ERROR - {'success': False, 'details': {'name': [ErrorDetail(string='This field may not be blank.', code='blank')]}} - No traceback available
2024-08-13 20:17:58,451 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 30
2024-08-13 20:18:55,204 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 30
2024-08-13 20:19:44,769 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 30
2024-08-13 21:14:23,507 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 30
2024-08-13 21:18:09,196 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 30
2024-08-13 21:44:44,596 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 30 - middleware - __call__ - ERROR - {'success': False, 'details': {'icon': [ErrorDetail(string='Not a valid string.', code='invalid')]}} - No traceback available
2024-08-13 21:51:30,485 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 30
2024-08-13 21:53:27,831 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 30
2024-08-13 21:54:37,046 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 30 - middleware - __call__ - ERROR - {'success': False, 'details': {'icon': [ErrorDetail(string='The submitted data was not a file. Check the encoding type on the form.', code='invalid')]}} - No traceback available
2024-08-13 22:03:02,840 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 30
2024-08-13 22:03:53,181 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 30
2024-08-13 22:04:19,834 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 30
2024-08-13 22:04:22,126 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 30
2024-08-13 22:05:35,105 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 30
2024-08-13 22:06:45,984 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 30
2024-08-13 22:07:33,948 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 30
2024-08-15 10:49:02,489 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponseNotFound' object has no attribute 'data' - line no: 30
2024-08-15 10:49:02,715 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 35 - middleware - __call__ - ERROR - 'HttpResponseNotFound' object has no attribute 'data' - line no: 30
2024-08-15 10:49:26,623 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 30 - middleware - __call__ - ERROR - {'detail': ErrorDetail(string='Authentication credentials were not provided.', code='not_authenticated')} - No traceback available
2024-08-15 11:01:18,552 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 48 - middleware - log_exception - ERROR - 'HttpResponseNotFound' object has no attribute 'data' - line no: 31
2024-08-15 11:01:45,043 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 48 - middleware - log_exception - ERROR - 'HttpResponseNotFound' object has no attribute 'data' - line no: 31
2024-08-15 11:02:15,943 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 48 - middleware - log_exception - ERROR - 'HttpResponseNotFound' object has no attribute 'data' - line no: 31
2024-08-15 11:02:52,043 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 48 - middleware - log_exception - ERROR - 'HttpResponseNotFound' object has no attribute 'data' - line no: 31
2024-08-15 11:03:22,668 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 48 - middleware - log_exception - ERROR - 'HttpResponseNotFound' object has no attribute 'data' - line no: 31
2024-08-15 11:04:32,548 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 48 - middleware - log_exception - ERROR - 'HttpResponseNotFound' object has no attribute 'data' - line no: 31
2024-08-15 12:48:01,866 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 48 - middleware - log_exception - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 31
2024-08-15 12:49:08,148 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 48 - middleware - log_exception - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 31
2024-08-15 12:53:30,727 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 43 - middleware - log_error - ERROR - {'detail': ErrorDetail(string='No App matches the given query.', code='not_found')} - No traceback available
2024-08-15 12:53:48,720 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 43 - middleware - log_error - ERROR - {'detail': ErrorDetail(string='No App matches the given query.', code='not_found')} - No traceback available
2024-08-15 12:54:41,231 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 43 - middleware - log_error - ERROR - {'detail': ErrorDetail(string='No App matches the given query.', code='not_found')} - No traceback available
2024-08-15 12:54:49,274 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 43 - middleware - log_error - ERROR - {'detail': ErrorDetail(string='No App matches the given query.', code='not_found')} - No traceback available
2024-08-15 12:58:55,210 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 43 - middleware - log_error - ERROR - {'success': False, 'details': {'icon': [ErrorDetail(string='The submitted data was not a file. Check the encoding type on the form.', code='invalid')]}} - No traceback available
2024-08-15 12:59:27,823 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 43 - middleware - log_error - ERROR - {'success': False, 'details': {'icon': [ErrorDetail(string='The submitted data was not a file. Check the encoding type on the form.', code='invalid')]}} - No traceback available
2024-08-15 13:00:45,113 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 43 - middleware - log_error - ERROR - {'success': False, 'details': {'icon': [ErrorDetail(string='The submitted data was not a file. Check the encoding type on the form.', code='invalid')]}} - No traceback available
2024-08-15 13:05:05,538 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 43 - middleware - log_error - ERROR - {'success': False, 'details': {'icon': [ErrorDetail(string='The submitted data was not a file. Check the encoding type on the form.', code='invalid')]}} - No traceback available
2024-08-15 13:05:22,783 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 43 - middleware - log_error - ERROR - {'success': False, 'details': {'icon': [ErrorDetail(string='The submitted data was not a file. Check the encoding type on the form.', code='invalid')]}} - No traceback available
2024-08-15 13:05:52,275 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 43 - middleware - log_error - ERROR - {'success': False, 'details': {'icon': [ErrorDetail(string='The submitted data was not a file. Check the encoding type on the form.', code='invalid')]}} - No traceback available
2024-08-15 13:07:43,515 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 43 - middleware - log_error - ERROR - {'success': False, 'details': {'icon': [ErrorDetail(string='The submitted data was not a file. Check the encoding type on the form.', code='invalid')]}} - No traceback available
2024-08-15 13:11:27,146 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 43 - middleware - log_error - ERROR - {'success': False, 'details': {'icon': [ErrorDetail(string='The submitted data was not a file. Check the encoding type on the form.', code='invalid')]}} - No traceback available
2024-08-15 13:11:49,744 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 43 - middleware - log_error - ERROR - {'success': False, 'details': {'icon': [ErrorDetail(string='The submitted data was not a file. Check the encoding type on the form.', code='invalid')]}} - No traceback available
2024-08-15 13:12:15,918 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 43 - middleware - log_error - ERROR - {'success': False, 'details': {'icon': [ErrorDetail(string='The submitted data was not a file. Check the encoding type on the form.', code='invalid')]}} - No traceback available
2024-08-15 13:13:07,739 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 43 - middleware - log_error - ERROR - {'success': False, 'details': {'icon': [ErrorDetail(string='The submitted data was not a file. Check the encoding type on the form.', code='invalid')]}} - No traceback available
2024-08-15 13:13:51,640 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 43 - middleware - log_error - ERROR - {'success': False, 'details': {'icon': [ErrorDetail(string='The submitted data was not a file. Check the encoding type on the form.', code='invalid')]}} - No traceback available
2024-08-15 13:15:08,328 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 43 - middleware - log_error - ERROR - {'success': False, 'details': {'icon': [ErrorDetail(string='The submitted data was not a file. Check the encoding type on the form.', code='invalid')]}} - No traceback available
2024-08-15 13:16:55,280 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 43 - middleware - log_error - ERROR - {'success': False, 'details': {'icon': [ErrorDetail(string='The submitted data was not a file. Check the encoding type on the form.', code='invalid')]}} - No traceback available
2024-08-15 13:25:26,283 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 43 - middleware - log_error - ERROR - {'detail': ErrorDetail(string='No App matches the given query.', code='not_found')} - No traceback available
2024-08-15 13:38:23,614 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 48 - middleware - log_exception - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 31
2024-08-15 13:48:37,558 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 43 - middleware - log_error - ERROR - {'detail': ErrorDetail(string='No App matches the given query.', code='not_found')} - No traceback available
2024-08-15 13:53:25,372 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 43 - middleware - log_error - ERROR - {'success': False, 'details': {'icon': [ErrorDetail(string='The submitted data was not a file. Check the encoding type on the form.', code='invalid')]}} - No traceback available
2024-08-15 13:53:52,693 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 48 - middleware - log_exception - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 31
2024-08-15 13:55:25,351 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 43 - middleware - log_error - ERROR - {'success': False, 'details': {'icon': [ErrorDetail(string='The submitted data was not a file. Check the encoding type on the form.', code='invalid')]}} - No traceback available
2024-08-15 15:31:38,908 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 48 - middleware - log_exception - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 31
2024-08-15 15:51:44,214 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 48 - middleware - log_exception - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 31
2024-08-15 15:52:30,676 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 48 - middleware - log_exception - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 31
2024-08-15 15:52:34,482 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 48 - middleware - log_exception - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 31
2024-08-15 15:52:58,678 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 48 - middleware - log_exception - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 31
2024-08-15 17:33:50,622 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 48 - middleware - log_exception - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 31
2024-08-15 17:35:45,368 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 48 - middleware - log_exception - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 31
2024-08-15 18:29:02,801 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 48 - middleware - log_exception - ERROR - 'HttpResponseNotFound' object has no attribute 'data' - line no: 31
2024-08-15 19:12:27,725 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 43 - middleware - log_error - ERROR - {'detail': ErrorDetail(string='Given token not valid for any token type', code='token_not_valid'), 'code': ErrorDetail(string='token_not_valid', code='token_not_valid'), 'messages': [{'token_class': ErrorDetail(string='AccessToken', code='token_not_valid'), 'token_type': ErrorDetail(string='access', code='token_not_valid'), 'message': ErrorDetail(string='Token is invalid or expired', code='token_not_valid')}]} - No traceback available
2024-08-15 19:12:28,131 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 48 - middleware - log_exception - ERROR - 'HttpResponseNotFound' object has no attribute 'data' - line no: 31
2024-08-15 19:12:44,656 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 43 - middleware - log_error - ERROR - {'detail': ErrorDetail(string='Given token not valid for any token type', code='token_not_valid'), 'code': ErrorDetail(string='token_not_valid', code='token_not_valid'), 'messages': [{'token_class': ErrorDetail(string='AccessToken', code='token_not_valid'), 'token_type': ErrorDetail(string='access', code='token_not_valid'), 'message': ErrorDetail(string='Token is invalid or expired', code='token_not_valid')}]} - No traceback available
2024-08-15 19:30:00,377 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 43 - middleware - log_error - ERROR - {'detail': ErrorDetail(string='Given token not valid for any token type', code='token_not_valid'), 'code': ErrorDetail(string='token_not_valid', code='token_not_valid'), 'messages': [{'token_class': ErrorDetail(string='AccessToken', code='token_not_valid'), 'token_type': ErrorDetail(string='access', code='token_not_valid'), 'message': ErrorDetail(string='Token is invalid or expired', code='token_not_valid')}]} - No traceback available
2024-08-15 20:42:54,864 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 47 - middleware - log_error - ERROR - {'detail': ErrorDetail(string='Given token not valid for any token type', code='token_not_valid'), 'code': ErrorDetail(string='token_not_valid', code='token_not_valid'), 'messages': [{'token_class': ErrorDetail(string='AccessToken', code='token_not_valid'), 'token_type': ErrorDetail(string='access', code='token_not_valid'), 'message': ErrorDetail(string='Token is invalid or expired', code='token_not_valid')}]} - No traceback available
2024-08-15 21:13:47,125 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 58 - middleware - log_exception - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 35
2024-08-16 00:14:07,838 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 58 - middleware - log_exception - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 35
2024-08-16 00:16:22,679 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 58 - middleware - log_exception - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 35
2024-08-16 00:16:25,012 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 58 - middleware - log_exception - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 35
2024-08-16 14:17:46,024 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 58 - middleware - log_exception - ERROR - 'HttpResponseNotFound' object has no attribute 'data' - line no: 35
2024-08-16 14:17:46,227 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 58 - middleware - log_exception - ERROR - 'HttpResponseNotFound' object has no attribute 'data' - line no: 35
2024-08-16 17:42:19,292 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 61 - middleware - log_exception - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 38
2024-08-16 17:51:39,244 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 61 - middleware - log_exception - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 38
2024-08-16 18:18:55,551 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 61 - middleware - log_exception - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 38
2024-08-16 18:20:47,871 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 61 - middleware - log_exception - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 38
2024-08-17 01:26:07,597 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 53 - middleware - log_error - ERROR - {'detail': ErrorDetail(string='Given token not valid for any token type', code='token_not_valid'), 'code': ErrorDetail(string='token_not_valid', code='token_not_valid'), 'messages': [{'token_class': ErrorDetail(string='AccessToken', code='token_not_valid'), 'token_type': ErrorDetail(string='access', code='token_not_valid'), 'message': ErrorDetail(string='Token is invalid or expired', code='token_not_valid')}]} - No traceback available
2024-08-17 01:30:09,215 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 53 - middleware - log_error - ERROR - {'detail': ErrorDetail(string='Given token not valid for any token type', code='token_not_valid'), 'code': ErrorDetail(string='token_not_valid', code='token_not_valid'), 'messages': [{'token_class': ErrorDetail(string='AccessToken', code='token_not_valid'), 'token_type': ErrorDetail(string='access', code='token_not_valid'), 'message': ErrorDetail(string='Token is invalid or expired', code='token_not_valid')}]} - No traceback available
2024-08-17 01:32:31,307 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 61 - middleware - log_exception - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 38
2024-08-17 01:44:33,029 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 61 - middleware - log_exception - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 38
2024-08-17 01:45:20,654 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 61 - middleware - log_exception - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 38
2024-08-17 01:47:19,250 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 61 - middleware - log_exception - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 38
2024-08-17 01:47:39,182 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 61 - middleware - log_exception - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 38
2024-08-17 02:16:32,384 - /home/varsha/Documents/Projects/nextgrowth/app_point_system1/app_point_system/middleware.py - 61 - middleware - log_exception - ERROR - 'HttpResponse' object has no attribute 'data' - line no: 38
